src/components/regions/MainSelectors.svelte

```diff
@@ -14,6 +14,12 @@ import {
 
 const OFFSET = 10;
 const COMPACT = true;
+
+let processDimension;
+$: if ($probe.info && $probe.info.calculated.seen_in_processes) {
+    processDimension = productConfig[$store.product].dimensions.process;
```

Comment:
You don't need to change anything here because this PR will probably merge before mine, but just something to be aware of for the future: in the future, this will probably be named:

```javascript
productConfig[$store.product].dimensions.productDimensions.process
```

See #687.

Reply:
Also:

```
  20:1   error  Expected indentation of 2 spaces but found 4         indent
```